nbconvert/filters/markdown.py

```diff
@@ -56,20 +54,37 @@ def markdown2latex(source, markup='markdown', extra_args=None):
     out : string
       Output as returned by pandoc.
     """
-    return pandoc(source, markup, 'latex', extra_args=extra_args)
+    warnings.warn("`markdown2latex` is deprecated in favor of " 
```

Comment:
I would prefer to *not* deprecate these names, as they are as clear as and more concise than the pandoc call with arguments. I'd just leave them as convenient wrappers.